Shared/Supporting Files/Views/WebView.swift

```diff
@@ -41,19 +41,18 @@ struct WebView: UIViewRepresentable {
     }
     
     class Coordinator: NSObject, WKNavigationDelegate {
-        func webView(
-            _ webView: WKWebView,
-            decidePolicyFor navigationAction: WKNavigationAction,
-            decisionHandler: @escaping (WKNavigationActionPolicy) -> Void
-        ) {
+        func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction) async -> WKNavigationActionPolicy {
             switch navigationAction.navigationType {
             case .linkActivated:
-                if let url = navigationAction.request.url, UIApplication.shared.canOpenURL(url) {
-                    UIApplication.shared.open(url)
+                if let url = navigationAction.request.url,
+                   await UIApplication.shared.canOpenURL(url) {
```

Comment:
For some reason the compiler expects that it is async. I'm guessing because it translates the function with the  completion handler to an async method. Even though the completion handler is optional and a default argument of `nil` is provided:

[uiapplication](https://developer.apple.com/documentation/uikit/uiapplication/1648685-open)